README.md

```diff
@@ -2,3 +2,39 @@ Instant Share
 =============
 
 Instantly share images, videos, and other files on the web.
+
+Features
+--------
+
+-	**Instant links**
+	-	When you share via Instant Share, you get a shareable link in your clipboard right away, so you can paste it in a conversation, in a forum post, or anywhere. That link is usable right away because Instant Share streams both the uploads and downloads in the background.
+-	**Share any file type**
+	-	You can share any image, video or other file types, including .html, .css, .txt and get a direct link to it. If possible, the files will be displayed in browser. Most browsers support streaming video downloads, making Instant Share the quickest way to share a video.
+
+Instant Share consists of a server and client.
+
+The client runs in your operating system's tray bar for quick access. The server is responsible for accepting uploads and serving those files.
+
+Platforms
+---------
+
+Instant Share client is fully supported on OS X. Linux and Windows clients are possible pending support being implemented in [`trayhost`](https://github.com/shurcooL/trayhost) package that Instant Share client uses for creating a tray icon.
+
+Instant Share server runs on OS X, Linux, Windows or any other platform that Go supports.
+
+Screenshots
+-----------
+
+![Screenshot 1](https://cloud.githubusercontent.com/assets/1924134/8891878/8dd7a024-32ee-11e5-8eca-1994f8503094.png)
+
+![Screenshot 2](https://cloud.githubusercontent.com/assets/1924134/8891877/8dc67272-32ee-11e5-9656-7f9749394ad3.png)
+
+API
```

Comment:
Let's not put useless stuff in the README as it takes away from what actually matters. API is for developers, not for users. They can dig it up themselves.